
After creating instance based on
could connect once but not again. This solved the issue (editing inbound rules of security group):

https://stackoverflow.com/questions/28952227/some-clients-fail-to-connect-to-postgresql-using-pgadmin
